lib/vsc/administration/vo.py

```diff
@@ -211,7 +208,18 @@ def _create_fileset(self, filesystem_name, path, parent_fileset=None, fileset_na
 
         The parent_fileset is used to support older (< 3.5.x) GPFS setups still present in our system
         """
-        self.gpfs.list_filesets()
+        fs_backend, _ = storage.load_operator()
```

Comment:
also method. (please avoid copy/pasting code)

Reply:
indeed, totally changed to use the new `StorageOperator` class in vsc-filesystems. See https://github.com/hpcugent/vsc-administration/pull/134/commits/557c3681738c7694cc80cfbef58535b6d0325e83